USEPA/EPA-Cyano-Web
# cyan-429-slide-show -> dev

Location details slideshow feature updates:
- Scrolls current displayed image's thumbnail into view.
- Updates cyano and image data.
- Loads latest image when location-details is initialized.
- Loads latest image data when existing location-details view.

To test:
- Select a location in "My Locations" to view its location details.
- Once thumbnail images are returned, the latest image will appear onto the map.
- Select "Slideshow" checkbox to initiate slideshow feature.
